server/appmanifest.go

```diff
@@ -0,0 +1,161 @@
+// Copyright (c) 2025-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package main
+
+import (
+	"archive/zip"
+	"bytes"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"strings"
+
+	"github.com/mattermost/mattermost-plugin-msteams-devsecops/assets"
+)
+
+const (
+	AppVersion          = "0.1.0"
+	AppID               = "3dfcc488-f38d-4867-bcba-84dfc8eb007b"
```

Comment:
If customers are self-deploying, I don't think we can hard-code `AppID`. This make the manifest generation less useful unless we can parameterize some of the inputs.